api/tests/routes/pro/test_offerers_addresses.py

```diff
@@ -373,7 +426,7 @@ def test_creating_offerer_address_when_only_municipality_centroid_is_the_default
         # Fetch the user
         # Check permissions
         # Insert address
-        # Select offererAddress if exists
+        # Select OffererAddress is exists
```

Comment:
Ne voulais-tu pas laisser `if` ?
```suggestion
        # Select OffererAddress if exists
```